packages/nextjs/hooks/flashbotRecoveryBundle/useRecoveryProcess.ts

```diff
@@ -7,7 +9,6 @@ import { useAccount, usePublicClient, useWalletClient } from "wagmi";
 import { RecoveryTx } from "~~/types/business";
 import { RecoveryProcessStatus } from "~~/types/enums";
 import { getTargetNetwork } from "~~/utils/scaffold-eth";
-import { useShowError } from "./useShowError";
 
 interface IStartProcessPops {
   safeAddress: string;
```

Comment:
let infuraProvider:InfuraProvider | null=null;

Reply:
I tried this methodology and it gave me errors. I moved it back to useState and it works. Maybe I missed something and you can take a stab at it.